components/hab/static/template_init

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
```

Comment:
Right - no need to create the run hook for them, as the typical path is that you just set `pkg_svc_run` and have done with it.